aws-apigateway-py-routes/__main__.py

```diff
@@ -0,0 +1,47 @@
+import pulumi
+import pulumi_aws as aws
+import pulumi_aws_apigateway as apigateway
+import lambdas_
```

Comment:
Is this trailing underscore required in the file name?

Reply:
Seemed like it broke without it - think lambda is a reserved keyword in python.

Reply:
But not `lambdas` I guess?

Reply:
Yup - the suffix can be removed fine 👍